docs/frontend/styleguide.md

```diff
@@ -0,0 +1,45 @@
+# Shopsys Framework styleguide
+
+## Documentation
+styleguide - tool to make creating and maintaining styleguides easy for Shopsys Framework designs.
+Contains all html elements necessary to create new design.
+
+## Installation to running SSFW project
+You need to have Shopsys Framework installed in developer mode according to our installation guide
```

Comment:
Link to installation guide would be nice

Reply:
Added